### PR TITLE
docs: list Awesome grammY in README

### PR DESCRIPTION
**Pull request by KnorpelSenf**
As requested in https://t.me/grammyjs/82935

### CODE REVIEW

**Comment by roziscoding** [comment]
LGTM, but is CI supposed to break?

**Review by roziscoding — APPROVED** [review]
LGTM :)

File: README.md
Comment (roj1512) [inline README.md:97]:
```suggestion
Helpful if you want to see some real-world usage.
```

**Review by roj1512 — APPROVED** [review]
LGTM.

**Comment by KnorpelSenf** [comment]
Hehe :D

Will merge after CI is fixed. This will likely be done in a dedicated PQ.

**Comment by KnorpelSenf** [comment]
Waiting for a review of #284.

**Pull request merged**
